marklogic-data-hub/src/test/java/com/marklogic/hub/EndToEndFlowTests.java

```diff
@@ -187,6 +208,7 @@ private JsonNode validateUserModules() {
         DataHub dataHub = getDataHub();
         List<DynamicTest> tests = new ArrayList<>();
 
+        // flows before a structured change get updated. this ru
```

Comment:
comment missing part of the sentence

Reply:
I'm going to remove it because I have no clue what I was typing there.